e2e-tests/src/test/treasury.rs

```diff
@@ -26,31 +33,26 @@ fn balance_info<C: ReadStorage>(connection: &C) -> (Balance, Balance) {
     (treasury_balance, issuance)
 }
 
-pub fn channeling_fee_and_tip(config: &Config) -> anyhow::Result<()> {
+pub async fn channeling_fee_and_tip(config: &Config) -> anyhow::Result<()> {
     let (transfer_amount, tip) = (1_000u128, 10_000u128);
-    let (connection, to) = setup_for_tipped_transfer(config, tip);
```

Comment:
actually checking whether tip is also transferred to treasury is quite important - can't we do it with subxt?